components/MyLayout.js

```diff
@@ -67,16 +66,10 @@ class MyLayout extends Component {
   }
 
   render () {
-    const { auth, children } = this.props
+    const { children } = this.props
     const { adminUser } = this.state
-    const { isAuthenticated, user } = auth
 
-    let contents
-    if (isAuthenticated && user && !user.email_verified) {
-      contents = <VerifyEmailScreen />
-    } else {
```

Comment:
I'm actually not so sure about moving the VerifyEmailScreen into the index page. If we were to add more pages, the user who has just signed up, could move about freely if this VerifyEmailScreen didn't block them from doing so. @evansiroky, was this your suggestion? Perhaps there's a better way?

Reply:
Moved email screen back to layout component.